rankone/config.py

```diff
@@ -13,9 +13,12 @@
 ]
 
 GAME_MODES_TO_PLAYERS = {'5v5': 10, '4v4': 8, '3v3': 6, '2v2': 4, '1v1': 2}
-
 MONITORED_GAME_MODES = ['4v4']
 
+STARTING_ELO = 1400
+STARTING_SIGMA = 200
+TRUESKILL_SETTINGS = {'beta': 500}  # this was chosen from simulations
```

Comment:
These 2 numbers allowed for players to reach 2k elo after 100 games with a decent-to-high win rate while still allowing players starting out to jump 40-50 elo per game. IMO that's a nice environment for encouraging players to grind pugs.